src/lv_interop.cc

```diff
@@ -44,6 +44,37 @@ namespace grpc_labview
 {
     grpc_labview::PointerManager<grpc_labview::gRPCid> gPointerManager;
 
+    //---------------------------------------------------------------------
+    //---------------------------------------------------------------------
+ 
+    // Initialize the static members
+    ProtoDescriptorString* ProtoDescriptorString::m_instance = nullptr;
+    std::string ProtoDescriptorString::descriptor_;
```

Comment:
how do we cleanup these things when server is stopped

Reply:
Added functionality to free memory when all servers stop